docs/dev-automated-aws-setup.md

```diff
@@ -0,0 +1,138 @@
+# Cloudigrade AWS setup with Ansible
+
+This is an alternative, automated workflow to the manual workflows described in [dev-houndigrade-cluster-setup.md](https://github.com/cloudigrade/cloudigrade/blob/master/docs/dev-houndigrade-cluster-setup.md) and [dev-cloudtrail-initial-setup.rst](https://github.com/cloudigrade/cloudigrade/blob/master/docs/dev-cloudtrail-initial-setup.rst). They are mutually exclusive.
+
+## Assumptions + Dependencies
+
+* Python executable in path must have boto3 importable
```

Comment:
Is the version or virtual env of the Python executable important?

Reply:
I was on python 2.7 and you were on python 3.6 (pretty sure), so AFAIK it is not important. I'm sure there could be some weird edge case.